spec/services/rapid_ready_for_decision/processor_selector_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe RapidReadyForDecision::ProcessorSelector do
+  subject { described_class.new(submission) }
+
+  let(:submission) { create(:form526_submission) }
```

Comment:
Is it possible to just `build` the form526 submission factory instead of `create`? I always ask, since there's a significant performance slowdown when using `create`.